alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -130,6 +131,23 @@ class AlertIndices(
         fun findingMapping() =
             AlertIndices::class.java.getResource("finding_mapping.json").readText()
 
+        @JvmStatic
+        fun getOrDefaultAlertIndex(monitor: Monitor): String {
+            var alertIndex = ALERT_INDEX
+            if (monitor.dataSources != null && monitor.dataSources.alertsIndex != null) {
```

Comment:
Do we need to check ``isEmpty`` as well?

Reply:
with changes i suggested, now `monitor.dataSources` can never be null. so the `if condition` simplifies